extensions/EXT/EXT_texture_mirror_clamp_to_edge.txt

```diff
@@ -0,0 +1,157 @@
+Name
+
+    EXT_texture_mirror_clamp_to_edge
+
+Name Strings
+
+    GL_EXT_texture_mirror_clamp_to_edge
+
+Contact
+
+    Christophe Riccio, Unity Technologies (christophe.riccio 'at' unity3d.com)
+
+Contributors
+
+    Contributors to ARB_texture_mirror_clamp_to_edge
+    Ian Romanick
+    Daniel Koch
+
+Notice
+
+    Copyright (c) 2017 The Khronos Group Inc. Copyright terms at
+        http://www.khronos.org/registry/speccopyright.html
+
+Status
+
+    Draft on September 7, 2017.
```

Comment:
Did you mean to switch this to Draft?

Reply:
I mean it but fixed. 
Probably I should have post it for review as draft :D